components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -481,6 +508,91 @@ - (void)preferredContentSizeDidChangeForChildContentContainer:(id<UIContentConta
   [self.view setNeedsLayout];
 }
 
+- (void)animateToPreferredContentHeight:(CGFloat)preferredContentHeight
+                           withDuration:(NSTimeInterval)duration
+                             completion:(void (^_Nullable)(BOOL))completion {
+  CGSize newPreferredContentSize =
+      CGSizeMake(CGRectGetWidth(self.view.bounds), preferredContentHeight);
+  if ([self isMobileLandscape]) {
+    self.contentViewController.preferredContentSize = newPreferredContentSize;
```

Comment:
can you not just return instead of setting the preferredContentSize?

Reply:
Add voiceover